From cb86643f363470bce947d028b187f043aaa9cbbd Mon Sep 17 00:00:00 2001
From: Sanjay R Mehta <sanju.mehta@amd.com>
Date: Wed, 21 Mar 2018 12:52:09 +0530
Subject: [PATCH 1354/4131] making kfd signal event limit macro to be page
 aligned

Error check in ifunction kfd_event_mmap() is failing without this
on Carrizo.
The macro value of KFD_SIGNAL_EVENT_LIMIT in 1.6.3 rocm was
(4096 +512) and by reverting it back rocm is functional on Carrizo.

Signed-off-by: Sanjay R Mehta <sanju.mehta@amd.com>
---
 include/uapi/linux/kfd_ioctl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/kfd_ioctl.h b/include/uapi/linux/kfd_ioctl.h
index de5367c..26e0abc 100644
--- a/include/uapi/linux/kfd_ioctl.h
+++ b/include/uapi/linux/kfd_ioctl.h
@@ -208,7 +208,7 @@ struct kfd_ioctl_dbg_wave_control_args {
 #define KFD_IOC_WAIT_RESULT_TIMEOUT	1
 #define KFD_IOC_WAIT_RESULT_FAIL	2
 
-#define KFD_SIGNAL_EVENT_LIMIT		4096
+#define KFD_SIGNAL_EVENT_LIMIT		(4096 + 512)
 
 struct kfd_ioctl_create_event_args {
 	uint64_t event_page_offset;	/* from KFD */
-- 
2.7.4

